integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.4.18
+0.4.20
```

Comment:
How does version number works ? Why the increase per 2.

Reply:
Overall rules are defined [here](https://github.com/ARMmbed/connectedhomeip/tree/master/integrations/docker/images/chip-build).
Here is some [example](https://github.com/project-chip/connectedhomeip/commit/c6f2e84512e096e10b8f4660bd009795ace0ddc2#diff-087ab3369ff49004266cdc5efba580a58b98e1396c5062728d9717da5eb6532b).

Yep, you're right - we should increase version to 0.4.19 - my mistake.

Reply:
Done.